cmds/ls/ls.go

```diff
@@ -17,21 +17,22 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"io"
 	"log"
 	"os"
 	"path/filepath"
 	"text/tabwriter"
+
+	flag "github.com/spf13/pflag"
 )
 
 var (
-	all     = flag.Bool("a", false, "show hidden files")
-	human   = flag.Bool("h", false, "human readable sizes")
-	long    = flag.Bool("l", false, "long form")
-	quoted  = flag.Bool("Q", false, "quoted")
-	recurse = flag.Bool("R", false, "equivalent to findutil's find")
+	all     = flag.BoolP("all", "a", false, "show hidden files")
+	human   = flag.BoolP("human", "h", false, "human readable sizes")
```

Comment:
Can you use the GNU long names?

```
--human-readable
--quote-name
--recursive
```

Reply:
I hate the gnu names but you are right, I have to do it.

Reply:
done